src/dmd/backend/cod3.d

```diff
@@ -3707,12 +3707,31 @@ void prolog_loadparams(ref CodeBuilder cdb, tym_t tyf, bool pushalloc, regm_t* n
 
             type *t = s.Stype;
             type *t2 = null;
-            if (tybasic(t.Tty) == TYstruct && config.exe != EX_WIN64)
+
+            tym_t tyb = tybasic(t.Tty);
+
+            // These logics are same as FuncParamRegs_alloc function at src/dmd/backend/cod1.d
```

Comment:
"This logic is the same as..."  In the dialect of English where I'm from, "logic" is an uncountable noun.